src/backend/parser/cypher_expr.c

```diff
@@ -489,25 +489,143 @@ static Node *transform_AEXPR_OP(cypher_parsestate *cpstate, A_Expr *a)
                            a->location);
 }
 
+//static Node *transform_AEXPR_IN(cypher_parsestate *cpstate, A_Expr *a)
```

Comment:
No need to comment out the code, just remove it. Unless you have a particular reason for leaving it in and commented out.

Reply:
I have corrected this change, thank you for pointing that out